docs/pages/server-access/guides.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Server Access Guides
+description: How to set up and configure Teleport Server Access for SSH and PAM
+---
+
+# Guides
+
+- [SSH](./guides/teleport-ssh.mdx)
+- [Using Teleport with PAM](./guides/ssh-pam.mdx)
+- [OpenSSH Guide](./guides/openssh-teleport.mdx)
```

Comment:
Let's rename it to `openssh.mdx`. We don't need `teleport` in the URL names, because our root contains teleport: `https://goteleport.com`